rsvim_core/src/state/ops.rs

```diff
@@ -85,3 +85,16 @@ pub enum Operation {
   /// Quit editor
   EditorQuit,
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+/// A set of possible motions when entering insert mode.
+pub enum InsertMotion {
+  /// keeps cursor at current position
+  Normal,
```

Comment:
I think we can name it "Keep", since "Normal" can lead people thinking about "normal mode" in vim.